core/rio/hdt/src/main/java/org/eclipse/rdf4j/rio/hdt/HDTBitmap.java

```diff
@@ -9,24 +9,47 @@
 
 import java.io.IOException;
 import java.io.InputStream;
+import java.io.OutputStream;
 import java.util.zip.CheckedInputStream;
+import java.util.zip.CheckedOutputStream;
 
 import org.eclipse.rdf4j.common.io.UncloseableInputStream;
+import org.eclipse.rdf4j.common.io.UncloseableOutputStream;
 
 /**
- *
+ * Bitmap
+ * 
+ * Just a series of 0 and 1 bits.
+ * 
+ * This is used in combination with a {@link org.eclipse.rdf4j.rio.hdt.HDTArray HDTArray} to encode triple parts.
+ * 
  * @author Bart Hanssens
  */
 class HDTBitmap extends HDTPart {
```

Comment:
Did you consider using the java bitset? https://docs.oracle.com/javase/8/docs/api/java/util/BitSet.html

Reply:
I considered it, but it does not make much of a difference at the moment. 
Bitset is a bit more sophisticated than what's needed here, and the HDTBitmap class moved in another direction by also parsing/writing and CRC-checking the bitmap block.

Though I should rephrase the javadoc to make this clear...